airflow-core/src/airflow/api_fastapi/core_api/datamodels/common.py

```diff
@@ -91,7 +92,7 @@ class BulkDeleteAction(BulkBaseAction[T]):
     """Bulk Delete entity serializer for request bodies."""
 
     action: Literal[BulkAction.DELETE] = Field(description="The action to be performed on the entities.")
-    entities: list[Union[str, T]] = Field(
+    entities: list[Union[str, BulkTaskInstanceBody]] = Field(
```

Comment:
Do we need a rebase or something to have that fixed? That shouldn't be here `T` is correct.